pkg/service/mockrecord/mockrecord.go

```diff
@@ -34,16 +37,22 @@ func (s *mockRecorder) MockRecord(path string, pid uint32, mockName string) {
 	models.SetMode(models.MODE_RECORD)
 	ys := yaml.NewYamlStore(path, path, "", mockName, s.logger)
 
+	//Initiate the telemetry.
```

Comment:
why is telemetry initialised here again ..? it is already done in main.go right..?

Reply:
yes, but how do I pass that variable to all the files in the codebase, if I create a global variable I will need to import main everywhere which will create the chance of a cyclic dependency.

Reply:
can't we send it through cmd.execute ..?